emva1288/unittests/test_results.py

```diff
@@ -300,3 +302,68 @@ def test_nans(self):
         self.assertIs(r.DSNU1288_DN(), np.nan)
 
         del r
+
+
+class TestRoutines(unittest.TestCase):
+
+    # instantiate a bayer filter and a wrong filter
+    bayer_filter = get_bayer_filter(0, 1, 1, 0, 1000, 1000)
+    wrong_filter = np.tile([[1, 1, 1, 0], [1, 0, 0, 1]], (500, 250))
+
+    # intantiate masked images based in the masks
+    img = np.abs(np.random.random([1000, 1000]))
+    wrong = np.ma.array(img, mask=wrong_filter)
+    masked = np.ma.array(img, mask=bayer_filter)
+
+    # save the filtered images, masked and with no mask
```

Comment:
the names img_filt, im_num, ma_filt, and ma_num are not clear, use something that reflects what they are a little bit better

Reply:
I put new names that are more logical